images/kubectl-build-deploy-dind/helmcharts/custom-ingress/templates/ingress.yaml

```diff
@@ -21,6 +21,13 @@ metadata:
     nginx.ingress.kubernetes.io/ssl-redirect: "true"
     ingress.kubernetes.io/ssl-redirect: "true"
     {{- end }}
+    kubernetes.io/tls-acme: "true"
```

Comment:
this probably creeped in here somehow

Reply:
ah, was in there before, probably got missed a removal during a merge conflict

Reply:
oh, actually....
https://github.com/amazeeio/lagoon/commit/48f84e13030e77d472076fb171aff39ab5270163